js/react.md

```diff
@@ -330,10 +314,15 @@ This is the GraphQL schema that you'll deploy to AWS AppSync. If you're familiar
 After the deployment is complete, open your `App.js` again and update the import to include both the `API` category and `graphqlOperation` method as follows:
 
 ```javascript
-import Amplify, { Analytics, Storage, API, graphqlOperation } from 'aws-amplify';
+import Amplify, {
+  Analytics,
+  Storage,
+  API,
+  graphqlOperation
+} from 'aws-amplify';
```

Comment:
P R E T T I E R

Reply:
Keeping it as split paragraphs for consistency with the rest instead of bullet point.